cmd/copy.go

```diff
@@ -1246,6 +1250,7 @@ func init() {
 	rootCmd.AddCommand(cpCmd)
 
 	// filters change which files get transferred
+	cpCmd.PersistentFlags().BoolVar(&raw.copyContents, "strip-top-dir", false, "copy everything under the source, not the source itself")
```

Comment:
>copy everything under the source, not the source itself [](start = 77, length = 55)

consider rephrase it be more user-friendly. "not the source itself" => something like "the top directory itself will not be replicated on the destination side"